### DISCUSSION TITLE
AMD GPU does not work after update IP ADAPTER ControlNet v1.1.431

### DISCUSSION THREAD

**Discussion by SunGreen777** [Mikubill/sd-webui-controlnet#2505] [General]
AMD GPU does not work after update IP ADAPTER ControlNet v1.1.431

**Comment by huchenlei** [comment]
See https://github.com/Mikubill/sd-webui-controlnet/issues/2506.

**Reply by huchenlei** [reply]
Update your extension should fix the issue.
